src/ws-tiles-chart/ws-tiles-chart.js

```diff
@@ -10,7 +10,12 @@ import {Tile} from './tile';
  * @property {Object} props.config Defines the color of each group of tiles
  * @property {string} props.title Title of the chart
  * @property {number} props.maxTileSize Defines the maximum size that the tile can be
```

Comment:
Is this maxTIleSize (also min...) in Pixel? I would recommend that for these two you also add "(in pixel)" at the end of the documentation so it is absolutely clear 😄